pages/UI.php

```diff
@@ -2007,7 +2007,7 @@ function DisplayNavigatorGroupTab($oP)
 			}
 		}
 
-		IssueLog::Error('UI.php operation='.$operation.', error='.$e->getMessage()."\n".$e->getTraceAsString());
+		IssueLog::Error('UI.php operation='.$operation.', error='.$e->getMessage()."\n".$e->getFullStackTraceAsString());
```

Comment:
Aren't you over-complicating thing?
```suggestion
		IssueLog::Error('UI.php operation='.$operation.', error='.$e->getMessage()."\n".$e);
```
Or this as the toString does give the message as well:
```suggestion
		IssueLog::Error('UI.php operation='.$operation.', error='.$e);
```

Reply:
Adding a new method allows us to have a clear PHPDoc explaining why we're doing this. And we're also not totally confident that Exception::toString will suit our needs in the future.

Reply:
Okay, can be solved for me.